_pages/en/capabilities.md

```diff
@@ -71,7 +30,66 @@ Average of "How much time" questions
   - [Section 10. Learning and Generative Culture](#section-10-learning-and-generative-culture)
   - [Section 11. Transformation and Transformational Leadership](#section-11-transformation-and-transformational-leadership)
 
-### SECTION 1. VERSION CONTROL AND CODE MAINTAINABILITY
+### WHAT IS AN ASSESSMENT TOOL
+
+An Assessment tool is a roadmap that helps to prospect self-improvement, uncover team challenges and opportunities through  capabilities, identifies priorities and tracks the progress.  It is easy to use and produce credible results. The tool provides the ability to review team performance and guide to achieve a desired outcome. It is a tool to drive technology performance improvement in the organization.  The Assessment tool is a survey that will help to measure team performance, understand strength and give recommendations on areas of improvement. It does not collect any data and the responses stay anonymous. The tool will show the areas where the competencies of your team do not reach the level required to reach team goals.  When gaps do not get addressed, they tend to get bigger, eventually causing teams to collapse. Provided recommendations for each section will fill the gaps in your team that allows them to grow and reach required goals. Received score and recommendations can be saved by a user in order to come back and take the survey again. When you compare the competencies defined in the first assessment to the results you received in the second, you will find how your team performance progressed. The process can be repeated until desired outcomes of team performance achieved.  
+
+#### What is the problem and solution  
+
+To manage today business environment challenges organizations need the ability to deliver software with speed and stability. It will help to improve technology and organizational performance.  
+
+#### Who is it for  
+
+IT teams in IITB but any Dev team can benefit from this tool.  
+
+#### What is the purpose of the Assessment tool  
+
+Based on the analysis of the answers, IT teams within the organization can see whether it is moving in the right direction. DevOps assessment tool helps to better understand strengths and needs, outline current maturity and identify a road map of improvement area.  
+For better outcomes use this path for continuous integration and delivery:  
+![A graphic with 4 circles demonstrating assessment cycle process. There are 4 circles connected with an arrow  in clockwise direction.
+The circle titles are: top "Assess"; right "Analyze"; bottom "Implement"; left "Improve".](assets/images/assessment_process_improvement.png){:height="100%" width="100%" style="display:block;margin-left:auto;margin-right:auto"}  
+
+#### Assessment tool process  
+
+Our process includes:  
+
+- Invitation to take a survey  
+- Choose a section to be assessed
```

Comment:
Could we say something like: **Choose a topic to self-assess** instead?

Reply:
yap! Changed as well! thanks G